src/renderers/shaders/ShaderChunk/common.glsl

```diff
@@ -4,6 +4,10 @@
 #define LOG2 1.442695
 #define EPSILON 1e-6
 
+#define square(a) a * a
```

Comment:
BTW the downside to a define as compared to a function is that you are relyiing upon the compiler to realize that the inner expression is squared.  For example ```square( 10 * log ( x ) )```, becomes: ```( 10 * log ( x ) ) * ( 10 * log ( x ) ) )```

Also please put brackets around "a" as well, so that this returns the right answer: ```square( 1 + 2 )```, because right now that expands into ```1 + 2 * 1 + 2```.